devops/scripts/benchmarks/main.py

```diff
@@ -509,6 +544,17 @@ def validate_and_parse_env_args(env_args):
         help="HIP device architecture",
         default=None,
     )
+    parser.add_argument(
+        "--unitrace",
+        action="store_true",
+        help="Unitrace tracing for sigle iteration of benchmarks",
+    )
+
+    parser.add_argument(
+        "--unitrace-inclusive",
+        action="store_true",
+        help="Regular run of benchmarks iterations and unitrace tracing in single additional run",
+    )
```

Comment:
please do `--unitrace inclusive` and `--unitrace` as a single option, not two separate ones. See how `--output-html` is done.